spec/requests/admin/configs_spec.rb

```diff
@@ -805,6 +805,22 @@
           }
           expect(Settings::UserExperience.public).to eq(is_public)
         end
+
+        it "updates display_in_directory to false" do
+          display_in_directory = false
+          post admin_settings_user_experiences_path, params: {
+            settings_user_experience: { display_in_directory: display_in_directory }
+          }
+          expect(Settings::UserExperience.display_in_directory).to eq(display_in_directory)
+        end
+
+        it "updates display_in_directory to true" do
```

Comment:
Since the default for this setting is "true" it seems like this test could easily be green even if the code doesn't actually work. That's why I prefer to use `change` assertions.

Reply:
Good call, updated!